src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -96,8 +97,23 @@ const getId = ({ pageType, service, variant, env }: GetIdProps) => {
       getIdFunction = () => pageType;
       break;
     case LIVE_PAGE:
+      getIdFunction = (path: string) => {
+        const isTipoId = isTipoIdCheck(path);
+        const isCpsId = isCpsIdCheck(path);
+        const id = isTipoId ? getTipoId(path) : getCpsId(path);
+        if (isTipoId) {
+          return `${id}`;
+        }
+        if (isCpsId) {
+          return `/${service}/live/${id}`;
+        }
+        return null;
```

Comment:
would this return null even without explicitly returning null if none of the if conditions were satisfied?